lib/spreedly/payment_methods/payment_method.rb

```diff
@@ -28,6 +28,8 @@ def self.new_from(xml_doc)
         return ThirdPartyToken.new(xml_doc)
       when 'google_pay'
         return GooglePay.new(xml_doc)
+      when 'apple_pay'
+        return ApplePay.new(xml_doc)
```

Comment:
Not an issue, just a Ruby newbie question about performance.  When ordering a "case" statement, is any thought given to ordering the "when"s so that the most likely hits are at the top, and the least likely are at the bottom?

Reply:
I've never really thought about it, I guess is you have a very large case statement it could have an effect but in general i would say the gains would be minimal.